src/osa/scripts/datasequence.py

```diff
@@ -285,6 +285,17 @@ def dl1_to_dl2(run_str: str) -> int:
     return analysis_step.rc
 
 
+def is_datasequence_running(run_str: str) -> bool:
+    """Return True if any datasequence jobs are running for the given run number."""
+    sacct_output = run_sacct()
+    sacct_info = get_sacct_output(sacct_output)
+    jobs_run = sacct_info[sacct_info["JobName"]=="LST1_"+run_str[:5]]
```

Comment:
running or pending, it should be for both (any job in the queue for that run number), right?

Reply:
yes

Reply:
then modify the docstring accordingly, maybe refer to submitted (running/pending) jobs instead of just running or jobs in the queue?